python/kserve/kserve/model.py

```diff
@@ -345,8 +345,10 @@ async def _http_predict(
                 predict_url, timeout=self.timeout, headers=predict_headers, content=data
             )
         except Exception as exc:
+            request_id = predict_headers.get("x-request-id", "unknown-request-id")
             logger.error(
                 f"Could not send a request to predictor at url {predict_url} "
+                f"for {request_id=} "
```

Comment:
```suggestion
                f"for {request_id} "
```

Reply:
Let's be consistent with predictive models here?
https://github.com/kserve/kserve/blob/9c6a6b8f0d8d431ebae828b132097cb8d4ea61ca/python/kserve/kserve/model.py#L161

Reply:
Updated the default value to `N.A.`

Reply:
@yuzisun the equals above was intentional (https://docs.python.org/3/whatsnew/3.8.html#f-strings-support-for-self-documenting-expressions-and-debugging). Is there a reason not to use it? We are at python 3.9 as far as i see.

Reply:
Ah ok ye deprecated 3.8